mods/blackjack/web/style.css

```diff
@@ -9,6 +9,34 @@ body {
   height: 100vh;
 }
 
+.player-box{
```

Comment:
Could you run this file through https://prettier.io to format it correctly?

Reply:
prettier is already setup on the codebase. only on ts files though